File: GTTS/INL.VettingService.Models/RejectVettingBatch_Param.cs
using INL.Services.Models;

namespace INL.VettingService.Models
{
    public class RejectVettingBatch_Param : BaseParam, IRejectVettingBatch_Param
    {
        public RejectVettingBatch_Param() : base() { }

        public long? VettingBatchID { get; set; }
        public string BatchRejectionReason { get; set; }
        public int? ModifiedByAppUserID { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/reject-vetting-batch_param.ts


import { IRejectVettingBatch_Param } from './ireject-vetting-batch_param';

export class RejectVettingBatch_Param implements IRejectVettingBatch_Param {
  
	public VettingBatchID?: number;
	public BatchRejectionReason: string = "";
  
	public ErrorMessages: string[] = null;
}


